### ISSUE TITLE
Issue with hotfix for w3c chromeOptions

### ISSUE THREAD

**Issue by riotCode** [facebook/php-webdriver#646]
### What are you trying to achieve? (Expected behavior)
Adapt acceptance testing with Codeception for ChromeDriver v75, using webdriver v1.7.0.

### What do you get instead? (Actual behavior)
In the code of the hotfix for w3c I get an error `toArray() called on Array` for `Facebook\WebDriver\Remote\RemoteWebDriver` line 112.
`if (!isset($chromeOptions->toArray()['w3c'])) {`

### How could the issue be reproduced? (Steps to reproduce)

Using Codeception pass capabilities to chromeOptions
```php
modules:
  enabled:
    - WebDriver:
        url: http://localhost:8000
        browser: chrome
        port: 9515
        restart: true
        capabilities:
          chromeOptions:
            args: ["--headless", "--disable-gpu", "--no-sandbox"]
```

### Details

* Php-webdriver version: 1.7.0
* PHP version: 7.3
* Operating system: Linux
* Browser used + version: ChromeDriver v75

I believe the cause is that the hotfix assumes the class `Facebook\WebDriver\Chrome\ChromeOptions` is always used, but before the hotfix it allowed passing an array instead.

**Comment by riotCode** [comment]
Ah, nevermind I see now it is fixed in main branch, just not released yet.

**Comment by OndraM** [comment]
Yes, fixed in 1.7.1. 

Ref. #642 .

**Comment by temptationdrops** [comment]
Still no substance
